source/features/edit-files-faster.tsx

```diff
@@ -11,9 +11,9 @@ import getDefaultBranch from '../github-helpers/get-default-branch';
 import onFileListUpdate from '../github-events/on-file-list-update';
 
 async function init(): Promise<void> {
-	const isPermalink = /Tag|Tree/.test(select('.branch-select-menu i')!.textContent!);
-	for (const fileIcon of select.all('.files :not(a) > .octicon-file')) {
-		const fileLink = fileIcon.closest('tr')!.querySelector<HTMLAnchorElement>('.js-navigation-open')!;
+	const isPermalink = /Tag|Tree/.test(select('[data-hotkey="w"] i')!.textContent!);
+	for (const fileIcon of select.all('.js-navigation-container .octicon-file')) {
```

Comment:
The `not` avoided wrapping the icon twice. It may not be necessary but it’s worth double-checking, perhaps `blam`ing the line to find discussions around it.

Reply:
It was there since the very first commit: [https://github.com/sindresorhus/refined-github/pull/1939/commits/5accac93731f530fe132ef30654a65afcbae51eb ](https://github.com/sindresorhus/refined-github/pull/1939/commits/5accac93731f530fe132ef30654a65afcbae51eb)

But I don't think it's necessary, because the file list will only be replaced as a whole (triggering the `onFileListUpdate` listener).